shell/common/api/features.cc

```diff
@@ -22,10 +21,6 @@ bool IsPDFViewerEnabled() {
   return BUILDFLAG(ENABLE_PDF_VIEWER);
 }
 
-bool IsRunAsNodeEnabled() {
-  return electron::fuses::IsRunAsNodeEnabled() && BUILDFLAG(ENABLE_RUN_AS_NODE);
```

Comment:
I think this the feature api is still needed so that tests can be excluded based on the fuse value ?

Reply:
we don't have infra to test different fuse values unfortunately